### DISCUSSION TITLE
How can I control the speed of the speech on pre-trained models?

### DISCUSSION THREAD

**Discussion by S4lXLV** [coqui-ai/TTS#2633] [General Q&A]
How can I control the speed of the speech on pre-trained models?
I'm using Python as they say in the docs simply something like this 

`tts = TTS(model_name="tts_models/de/thorsten/tacotron2-DDC", progress_bar=False, gpu=False)
tts.tts_to_file(text="Ich bin eine Testnachricht.", file_path=OUTPUT_PATH)
`

**Comment by eneszv** [comment]
Maybe you can speed up or slow down the speech later with `pydub`
